bootstrap/startup.json.default

```diff
@@ -29,6 +29,10 @@
             "/var/logs/blueos": {
                 "bind": "/var/logs/blueos",
                 "mode": "rw"
+            },
+            "/home/pi/userdata": {
+                "bind": "/home/pi/userdata",
```

Comment:
We should not bind the host computer with the username, we should use the root folder or a better path, maybe `/usr/blueos`

Reply:
Agreed with @patrickelectric, especially for future consistency with #1073